src/main/java/uk/gov/justice/hmpps/prison/service/SentenceEnvelopeService.kt

```diff
@@ -75,48 +63,36 @@ class SentenceEnvelopeService(
       offenderBooking.offender.birthDate,
       WordUtils.capitalizeFully(offenderBooking.offender.lastName),
       offenderBooking.location.id,
-      offenderBooking.alerts.stream().filter { obj: OffenderAlert -> obj.isActive }
-        .map { offenderAlert: OffenderAlert? -> OffenderAlertTransformer.transformForOffender(offenderAlert) }
-        .collect(Collectors.toList()),
+      offenderBooking.alerts.filter { it.isActive }
+        .map { OffenderAlertTransformer.transformForOffender(it) },
     )
-    val sentenceAdjustments =
-      offenderBooking.getSentenceAdjustments().stream().filter { obj: SentenceAdjustmentValues -> obj.isActive }
-        .toList()
-    val bookingAdjustments =
-      offenderBooking.getBookingAdjustments().stream().filter { obj: BookingAdjustment -> obj.isActive }
-        .toList()
-    val sentences = bookingService.getSentenceAndOffenceDetails(offenderBooking.bookingId).stream()
-      .filter { obj: OffenderSentenceAndOffences? -> Objects.nonNull(obj) }
-      .filter { sentence: OffenderSentenceAndOffences -> sentence.sentenceStatus == "A" }.toList()
-    val containsFine = sentences.stream().anyMatch { obj: OffenderSentenceAndOffences -> obj.isAFine }
-    val containsFixedTermRecall =
-      sentences.stream().anyMatch { obj: OffenderSentenceAndOffences -> obj.isFixedTermRecallType }
-    val offenderFinePaymentDtoList = getFinesIfRequired(containsFine, offenderBooking.bookingId)
-    val fixedTermRecallDetails = getFixedTermRecall(containsFixedTermRecall, offenderBooking.bookingId)
-    val sentenceCalcDates = bookingService.getBookingSentenceCalcDatesV1_1(offenderBooking.bookingId)
+    val sentenceAdjustments = offenderBooking.getSentenceAdjustments().filter { it.isActive }
+    val bookingAdjustments = offenderBooking.getBookingAdjustments().filter { it.isActive }
+    val bookingId = offenderBooking.bookingId
+
+    val sentences = offenderBooking.sentences.filter {
```

Comment:
remove call to `bookingService.getSentenceAndOffenceDetails` as we have the sentences in memory anyway.  This was verifying booking access and then calling `findByOffenderBooking_BookingId_AndCalculationType_CalculationTypeNotLikeAndCalculationType_CategoryNot`, hence the extra conditions on the filter